src/Wallabag/CoreBundle/Resources/views/themes/baggy/Entry/entries.html.twig

```diff
@@ -1,7 +1,11 @@
 {% extends "WallabagCoreBundle::layout.html.twig" %}
 
 {% block title %}
-    {% include "@WallabagCore/themes/common/Entry/_title.html.twig" %}
+    {% set params = {} %}
+    {% if tag is defined %}
+        {% set params = {'label': tag} %}
```

Comment:
Can this be named differently? since it only handle tag.label atm?

```twig
{% set currentTag = tag.label %}
....
{% include "@WallabagCore/themes/common/Entry/_title.html.twig" with {'currentTag': currentTag} %}
```